### PR TITLE
CHE-728

### PR DESCRIPTION
**Pull request by mmorhun**
CHE-728: Fix an error that occurred while writing line to the line consumer during che shutting down
Signed-off-by: Mykola Morhun mmorhun@codenvy.com

### CODE REVIEW

**Comment by mmorhun** [comment]
@evoevodin check please

File: wsmaster/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/MachineManager.java [hunk 1]
```diff
@@ -1011,24 +1011,14 @@ private void createLogsDir() {
...
-            interrupted = true;
-            executor.shutdownNow();
-        }
 
         try {
             for (MachineImpl machine : machineRegistry.getMachines()) {
                 try {
                     destroy(machine.getId(), false);
+                } catch (NotFoundException ignore) {
+                    // if machine was destroyed before, it is ok
```
Comment (voievodin) [inline wsmaster/che-core-api-machine/src/main/java/org/eclipse/che/api/machine/server/MachineManager.java]: I'd say that it's better to use comment like `it is ok, machine has been already destroyed`

**Comment by voievodin** [comment]
Please change the comment and :+1:  good job

**Comment by garagatyi** [comment]
LGTM

**Pull request merged**